app/Controllers/App.php

```diff
@@ -491,4 +491,41 @@ public static function termListUrl($taxonomy)
         }
         return get_permalink($page->ID);
     }
+
+    public static function isSameDomain($string1, $string2)
+    {
+      $url1 = parse_url($string1);
+      $url2 = parse_url($string2);
+      if (isset($url1['host']) && isset($url2['host'])) {
+        if ($url1['host'] == $url2['host']) {
+           return true;
+        }
+      }
+      return false;
+    }
```

Comment:
I think we can remove this function altogether if we set the href of the link to the current site to `/`. Then we can check to see if `$menuitem->url` === `/` instead of comparing domains.